public/css/app.css

```diff
@@ -0,0 +1,310 @@
+/* body */
+
+body {
+    background-color: #ccc;
+}
+
+/* container */
+
+.container {
+    position: relative;
```

Comment:
But I feel the indentation under "position" makes it easier to make changes later, because left/top/transform are all very pertinent to repositioning elements which happens frequently with a project like this.